bin/build-plugin-zip.sh

```diff
@@ -77,10 +77,17 @@ fi
 status "Gathering PHP dependencies... 🐿️"
 composer install --no-dev
 
-# Run the build.
-status "Generating build... 👷‍♀️"
-npm run build
-npm run docs
+# Build the Core files.
+status "Generating a Core build... 👷‍♀️"
+WC_ADMIN_PHASE=core npm run build
+
+# Make a Github release.
+status "Starting a Github release... 👷‍♀️"
```

Comment:
So this will create a GitHub release (or at least ask if you want to) every time we build a version of the feature plugin?

I feel like it might be better to have the core build step/GitHub release run as separate commands from the plugin build.

We could add an additional parameter to `npm run build:release` to pass if it should be core or plugin, and only run the GitHub stuff if core is passed.

We also can create additional zips with different features enabled: https://github.com/woocommerce/woocommerce-admin/blob/master/docs/feature-flags.md#building-custom-plugin-builds, which doesn't need to run the core/GitHub commands -- but would with this change.

Reply:
>We could add an additional parameter to npm run build:release to pass if it should be core or plugin, and only run the GitHub stuff if core is passed.

We could definitely do this. I figured making two releases each time is Ok because an update in core would be done by updating Core's `composer.json`. However, you've raised a great point, which is that we have different zips now (core, plugin, features like OBW) and maybe each release should be more explicit, like the one @joshuatf did https://github.com/woocommerce/woocommerce-admin/releases/tag/v0.21.0-onboarding.

How about if I incorporate the slug/feature commands such that a GH release is made automatically specific to that slug? 

For instance:

* `npm run build:release` creates the plugin configs for GH release `vx.x.x` and `woocomerce-admin.zip`.
* `npm run build:release -- --slug core` creates the core configs for GH release `vx.x.x-core` and `woocomerce-admin-core.zip`.
* `npm run build:release -- --slug onboarding --features '{"onboarding":true}'` creates the modified configs for GH release `vx.x.x-onboarding` and `woocomerce-admin-onboarding.zip`.

Reply:
I decided to do something else:

If a slug or feature set is designated, then I just skip the core release. However, I want to automate the creation of the release a la v0.21.0-onboarding.

Would that work?

Reply:
> If a slug or feature set is designated, then I just skip the core release. However, I want to automate the creation of the release a la v0.21.0-onboarding.

That sounds good to me. 👍I think it'll be useful to be able to do that without worrying about whats generated for core.